src/mlpack/methods/preprocess/preprocess_describe_main.cpp

```diff
@@ -85,6 +83,23 @@ PARAM_FLAG("row_major", "If specified, the program will calculate statistics "
     "across rows, not across columns.  (Remember that in mlpack, a column "
     "represents a point, so this option is generally not necessary.)", "r");
 
+/**
+ * Formats string similar to printf.
+ *
+ * @param format Format of string desired.
+ * @param args Args for inserting as parameter.
+ * @return Desired Formatted String.
+ */
+template<typename ... Args>
+std::string string_format(const std::string& format, Args ... args)
+{
+    size_t size = snprintf( nullptr, 0, format.c_str(), args ... ) + 1; // Extra space for '\0'
+    if( size <= 0 ){ throw std::runtime_error( "Error during formatting." ); }
+    std::unique_ptr<char[]> buf( new char[ size ] ); 
+    snprintf( buf.get(), size, format.c_str(), args ... );
+    return std::string( buf.get(), buf.get() + size - 1 ); // We don't want the '\0' inside
+}
```

Comment:
This function does not follow mlpack style. Would you correct the style?

Reply:
Ig I have done changes